.github/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Changes from release 2024/11 to 2024/12
 1. AIRAC (2412) - Corrected Luton (EGGW) Tug Release Point V orientation - thanks to @AdriTheDev (Callum Hicks)
+2. Enhancement - Add Manchester (EGCC) Taxiline Linking D and K - thanks to @rishab-alt
```

Comment:
Should be in [past tense](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#entry-summary-)